### ISSUE TITLE
HTTP Resolver fail

### ISSUE THREAD

**Issue by jpstroop** [loris-imageserver/loris#249]
@peichman-umd 

```
Traceback (most recent call last):
  File "/home/jstroop/workspace/loris/tests/resolver_t.py", line 278, in test_template_http_resolver
    self.app.resolver._web_request_url('a:foo.jpg'))
  File "/home/jstroop/workspace/loris/loris/resolver.py", line 461, in _web_request_url
    url = self.templates[prefix]['url'] % ident
TypeError: string indices must be integers, not str
```
This was hidden due to #247. Can you get a fix in?

**Comment by peichman-umd** [comment]
@jpstroop Looking at it now.

**Comment by jpstroop** [comment]
Thanks!

**Comment by peichman-umd** [comment]
Opened PR #250 that should fix this. It was a mixture of bad test data and the TemplateHTTPResolver not returning the expected `(url, options)` tuple in cases where the identifier couldn't be resolved.

**Comment by bcail** [comment]
Fixed by PR #250.
